kubernetes/k8s-listener.py

```diff
@@ -388,6 +439,88 @@ def main():
         if c != '\r' or c2 != '\n':
             raise "Expected CRLF, got %c%c" % (c, c2)
 
+def get_romana_hosts():
+    """
+    returns a list of romana host IPs
+    
+    Example: ["10.0.0.1", 10.1.0.1" ]
+    """
+
+    romana_hosts = []
+    r = requests.get(topology_url)
+    print 'Topology service returned %s' % r.content
+    hosts = simplejson.loads(r.content)
+
+    # romana_ip is a CIDR like 10.0.0.1/16
+    # we only want ip part
+    for host in hosts:
+        mask_idx = host["romana_ip"].index("/")
+        romana_hosts.append(host["romana_ip"][:mask_idx])
+    return romana_hosts
+
+# We want to receive json object as a POST.
+from BaseHTTPServer import BaseHTTPRequestHandler,HTTPServer
+from mimetools import Message
+from StringIO import StringIO
+PORT_NUMBER = 9630
+HTTP_Unprocessable_Entity = 422
+class AgentHandler(BaseHTTPRequestHandler):
+    def do_GET(self):
```

Comment:
What is this function used for? Does anyone use it? If we keep it here then we should say a bit more than 'Hello Get !'.